### DISCUSSION TITLE
🚀 Feature Request: add a /bin/sh button

### DISCUSSION THREAD

**Discussion by gaspoweredflea** [louislam/dockge#217] [Feature Request]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

UI Feature

### 🔖 Feature description

At the moment there is a button to access /bin/bash in the docker container.
Some containers don't have bash available, so it. would be most useful to add a second button for /bin/sh for those that don't!

### ✔️ Solution

Add a second UI button for /bin/sh

### ❓ Alternatives

Perhaps a drop down?

**Comment by louislam** [comment]
![image](https://github.com/louislam/dockge/assets/1336778/e8691345-120f-4757-970b-1cb427dd18cd)

You can switch to sh here

**Reply by gaspoweredflea** [reply]
Ah! Maybe time for that annual optometrist visit ;)
Thanks for the wonderful software!
